frontend/src/components/modal-compare/styles.module.scss

```diff
@@ -0,0 +1,84 @@
+.container {
+  position: fixed;
+  top: 20px;
+  right: 20px;
+  z-index: 10;
+}
+
+.content {
+  position: relative;
+  display: flex;
+  max-width: 400px;
+  padding: 25px 30px 30px;
+  background-color: var(--white);
+  border-top: 5px solid var(--blue100);
+  border-radius: 5px;
+}
+
+.close {
+  position: absolute;
+  top: 5px;
+  right: 20px;
+  padding: 0;
+  color: var(--gray400);
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 1;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  opacity: 0.4;
+
+  &:hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+}
+
+.icon {
+  margin-top: 10px;
+  margin-right: 8px;
+  padding: 0;
+}
+
+.button {
+  width: 20px;
+  height: 20px;
+}
+
+.message {
+  align-self: center;
+  padding-right: 20px;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 50%;
+}
+
+.carDescription {
+  max-height: 85px;
+  margin: 0;
+  margin-top: 10px;
+  overflow: hidden;
+  color: var(--gray400);
+  font-weight: 400;
+  font-size: 90%;
+  font-style: normal;
+  line-height: 17px;
+  opacity: 0.6;
+}
+
+.inline {
+  display: inline;
+}
+
+.carName {
+  display: inline-block;
+  color: var(--blue100);
+  text-transform: uppercase;
+}
+
+@media (max-width: 440px) {
+  .container {
+    right: 0;
+  }
+}
```

Comment:
Please add some side margin for the card when it's used on mobile. It should stick to the sides of the screen